Marlin/src/inc/Conditionals_LCD.h

```diff
@@ -522,6 +527,7 @@
 
 #if IS_ULTIPANEL && DISABLED(NO_LCD_MENUS)
   #define HAS_MARLINUI_MENU 1
+  #define HAS_BACK_ITEM 1
```

Comment:
It makes no sense to define this here, and then again –with different conditions– in `Conditionals_post.h`.